src/common/module/fcttre_mod.F90

```diff
@@ -31,20 +31,99 @@ MODULE FCTTRE_MOD
 !     ------------------------------------------------------------------
 
   USE PARKIND1, ONLY : JPIM, JPRB
-
-  USE YOMCST, ONLY : RG, RD, RCPD, RETV, RLVTT, RLSTT, RLMLT, RTT, RV, RA, RPI
-  USE YOETHF, ONLY : R2ES, R3LES, R3IES, R4LES, R4IES, R5LES, R5IES, &
+  IMPLICIT NONE
+  REAL(KIND=JPRB),PUBLIC, PROTECTED :: &
+ &                 RG, RD, RCPD, RETV, RLVTT, RLSTT, RLMLT, RTT, RV, RA, RPI,          &
+ &                 R2ES, R3LES, R3IES, R4LES, R4IES, R5LES, R5IES,                     &
```

Comment:
I'm not 100% convinced that this is / will be neutral for all the other variants. Is there a way to make this an optional mode ? Or should we revisit these in a separate PR to test and evaluate in isolation - this could have quite an impact on all the other variants, if it isn't explicitly protected.